.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,31 @@
+name: GstVkVideoParsers CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Dependecies
+      run: |
+        sudo apt update
+        sudo apt install pkg-config
+        sudo apt install libglib2.0-dev libjson-glib-dev libncursesw5-dev
```

Comment:
afaik, there's no need for these manually installed dependencies

Reply:
removed